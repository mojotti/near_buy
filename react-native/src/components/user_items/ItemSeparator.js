import React, { Component } from 'react';
import { View } from 'react-native';
import { styles } from '../../static/styles/ListViewItemStyle';

export default class ItemSeparator extends Component {
  render() {
    return <View style={styles.separator} />;
  }
}
