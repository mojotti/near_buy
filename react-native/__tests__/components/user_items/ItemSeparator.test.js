import React from 'react';
import { shallow } from 'enzyme';

import ItemSeparator from '../../../src/components/user_items/ItemSeparator';

describe('<ItemSeparator />', () => {
  test('renders correctly', () => {
    expect(shallow(<ItemSeparator />)).toMatchSnapshot();
  });
});
