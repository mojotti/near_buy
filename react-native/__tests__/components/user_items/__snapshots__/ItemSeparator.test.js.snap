// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemSeparator /> renders correctly 1`] = `
<Component
  style={
    Object {
      "alignSelf": "center",
      "borderBottomColor": "#858585",
      "borderBottomWidth": 0.5,
      "justifyContent": "center",
      "marginBottom": 1,
      "width": 697.5,
    }
  }
/>
`;
